lib/tag/vdom.js

```diff
@@ -38,10 +38,11 @@ function mountTo(root, tagName, opts) {
 
 }
 
-riot.tag = function(name, html, css, fn) {
+riot.tag = function(name, html, css, attrs, fn) {
+  if (typeof attrs == 'function') {fn = attrs; attrs=css}
```

Comment:
Hi @rsbondi. It seems breaking the backward compatibility. Without re-compilation, I've got an error.
How about this?

```javascript
riot.tag = function(name, html, css, attrs, fn) {
  if (typeof attrs == 'function') { fn = attrs; attrs = '' } // < v2.0.16
  if (typeof css == 'function') { fn = css; attrs = '' } // < v2.0.0.10
  else if (css) injectStyle(css)
  tagImpl[name] = { name: name, tmpl: html, attrs: attrs, fn: fn }
  return name
}
```